IBAnimatable/PresenterManager.swift

```diff
@@ -19,12 +19,12 @@ class PresenterManager {
   }
   
   // MARK: - Private
-  private var cache = [TransitionAnimationType: Presenter]()
+  private var cache = [String: Presenter]()
```

Comment:
Since `stringValue` is a var, we can't directly use `TransitionAnimationType.stringValue`, or I'm missing something. Any suggestions ? :)